src/pages/DetailPage/Components/ProductInfo/ProductInfo.js

```diff
@@ -1,29 +1,64 @@
 import React, { Component } from "react";
 import "./ProductInfo.scss";
-import Options from "./Option/Options";
+
 import ProductRecommendation from "../../../MainPage/components/ProductRecommendation/ProductRecommendation";
 
 class ProductInfo extends Component {
-  state = {
-    chosenProduct: [
-      { option: "비올라(Viola) (재고36개)", num: 1, price: "20000원" }
-    ]
-  };
+  constructor() {
+    super();
+    this.state = {
+      price: "",
+      chosenProduct: []
+    };
+  }
+
+  componentDidMount() {
+    const { price } = this.props.productInfo;
+    this.setState({ price: price });
```

Comment:
Key와 value명이 같을 때에는 아래와 같이 작성해주실 수 있습니다:)
```suggestion
    this.setState({ price });
```

Reply:
wow!